internal/pkg/scaffold/gopkgtoml_test.go

```diff
@@ -48,31 +48,31 @@ required = [
 
 [[override]]
   name = "k8s.io/api"
-  # revision for tag "kubernetes-1.13.1"
-  revision = "05914d821849570fba9eacfb29466f2d8d3cd229"
+  # revision for tag "kubernetes-1.14.1"
```

Comment:
Could we bump the version on these ones?
```suggestion
  version = "kubernetes-1.14.1"
```

Reply:
These need to stay pinned to a revision because `dep` does not resolve them correctly.